packages/app/src/sandbox/eval/presets/vue-cli/index.js

```diff
@@ -127,6 +128,9 @@ export default function initialize() {
   vuePreset.registerTranspiler(module => /\.vue$/.test(module.path), [
     { transpiler: vueTranspiler },
   ]);
+  vuePreset.registerTranspiler(module => /\.coffee$/.test(module.path), [
+    { transpiler: coffeeTranspiler },
+  ]);
```

Comment:
I originally added this `coffee` transpiler to the end of the file, but realized that it was not being executed since the level or declaration determines its priority and it was lower than the `raw-loader`.